### ISSUE TITLE
Error message after recording started.

### ISSUE THREAD

**Issue by ryanek011** [voidtemp/OpenXStreamLoader#24]
recording worked well on Cahturbate. From today i get error message after a few seconds after hitting the start button. Status column shows Error. Any ideas?
Thanks.

**Comment by voidtemp** [comment]
Hello
Could you please say what's in Streamlink log (right click on task)?

**Comment by ryanek011** [comment]
It's been working well again. I had issues only yesterday for some reason. Thanks for help anyway. If the error message re-appears i'll post the Streamlink log.

**Comment by Ta8Success** [comment]
this could happen when you're trying to record a stream but the quality isn't supported. 
like setting the "quality to 1080p" even though the streamer only streams at a 720p.

**Comment by ryanek011** [comment]
Yes. That's exactly the reason. The streamlink log says that the 1080p stream is not found and the best is 720p for example. Then the quality needs to be manually set to lower figure and it works fine then.
